### DISCUSSION TITLE
how many particles should be used for training model?

### DISCUSSION THREAD

**Discussion by jianhaoc** [tbepler/topaz#105] [General]
Hi, I wonder how many particles at least should be used for training model? Are there any difference among proteins of different symmetries? Thank you!

**Comment by Guillawme** [comment]
The typical recommendation (at least last time I heard @alexjnoble give a seminar about Topaz) is to manually pick about 1000 particles, making sure that they are well centered and you picked as many distinct orientations you could see.
